### PR TITLE
CORTX-29712: Motr support bundle is not generating after enabling fil…

### PR DESCRIPTION
**Pull request by papan-singh**
…ter flags.

Signed-off-by: Papan Kumar Singh <papan.kumarsingh@seagate.com>

# Problem Statement
- Problem statement

# Design
-  For Bug, Describe the fix here.
-  For Feature, Post the link for design

# Coding
   Checklist for Author
-  [ ] Coding conventions are followed and code is consistent

# Testing 
  Checklist for Author
- [ ] Unit and System Tests are added
- [ ] Test Cases cover Happy Path, Non-Happy Path and Scalability
- [ ] Testing was performed with RPM

# Impact Analysis
  Checklist for Author/Reviewer/GateKeeper
- [ ] Interface change (if any) are documented
- [ ] Side effects on other features (deployment/upgrade)
- [ ] Dependencies on other component(s)

# Review Checklist 
  Checklist for Author
- [x] JIRA number/GitHub Issue added to PR
- [x] PR is self reviewed
- [x] Jira and state/status is updated and JIRA is updated with PR link
- [x] Check if the description is clear and explained

# Documentation
  Checklist for Author
- [ ] Changes done to WIKI / Confluence page / Quick Start Guide

### CODE REVIEW

**Review by gargpalak048 — APPROVED** [review]
LGTM

File: scripts/install/opt/seagate/cortx/motr/libexec/m0reportbug-bundler [hunk 1]
```diff
@@ -116,27 +124,27 @@ while true; do
          --binlogs)
             #should only accept either true or false
             rc=$(echo $2 | sed -e 's/,/\n/g') ;
-            if [[ "$rc" != "true" &&  "$rc" != "false" ]]; then
+            if [[ "$rc" != "true" && "$rc" != "True" && "$rc" != "false" && "$rc" != "False" ]]; then
```
Comment (cdeshmukh) [inline scripts/install/opt/seagate/cortx/motr/libexec/m0reportbug-bundler:127]: if [ "${rc,,}" != "true" ]; 

similar changes everywhere else can be applied

File: scripts/install/opt/seagate/cortx/motr/libexec/m0reportbug-bundler [hunk 2]
```diff
@@ -98,16 +98,24 @@ while true; do
...
+            if [[ $unit =~ ^[a-zA-Z]+$ ]]; then
+                sb_size_limit="${size_limit:0:$str_len - 2}"
+                sb_size_limit=${sb_size_limit%.*}
+            else
+                unit=${size_limit: -1}
+                sb_size_limit="${size_limit:0:$str_len - 1}"
+                sb_size_limit=${sb_size_limit%.*}
+            fi
+
+            if [[ "$unit" != "G" && "$unit" != "GB" && "$unit" != "M" && "$unit" != "MB" && "$unit" != "K" && "$unit" != "KB" ]]; then
```
[the remark this answers is not in the document]
Comment (papan-singh) [reply scripts/install/opt/seagate/cortx/motr/libexec/m0reportbug-bundler:112]: incorporated

File: scripts/install/opt/seagate/cortx/motr/libexec/m0reportbug-bundler (hunk 1 above)
Comment (papan-singh) [reply scripts/install/opt/seagate/cortx/motr/libexec/m0reportbug-bundler]: incorporated

**Comment by papan-singh** [comment]
tested on k8s env
[root@cortx-data-headless-svc-ssc-vm-g4-rhev4-1293 ~]# /opt/seagate/cortx/motr/libexec/m0reportbug-bundler -b bundle-logs-cortx-data-ssc-vm-g4-rhev4-1209-6554467f46-qc2d4-2022-03-25_00-17_7f4f141538794027a07e56de77107aa8_motr -t /var/cortx/support_bundle/bundle-logs-cortx-data-ssc-vm-g4-rhev4-1209-6554467f46-qc2d4-2022-03-25_00-17/ -c yaml:///etc/cortx/cluster.conf -s io --duration P5D --size_limit 833.3333333333334MB --binlogs false --coredumps "False" --stacktrace "true"
probes... OK
misc_files... OK
backtraces_live... OK
coredump_live... *WARNING* ## live core are not collected on kvm ##
OK
backtraces_cores... OK
m0traces... OK
addb... OK
addb-files... OK
systemd_journal... No journal files were found.
OK
sysrq_info... *WARNING* /proc/sysrq-trigger is read-only. Won't be able to collect system information.
OK
*WARNING* /var/crash/: no such directory
*WARNING* /var/log/crash/: no such directory
*WARNING* /var/log/seagate/motr/tmpspace/: no such directory
m0traces /opt/seagate/cortx/motr/libexec... OK
m0traces /etc/cortx/log/motr/c2429b70323846f0a44052f9a6572a24/trace... OK (0m47s)
addb-files /etc/cortx/log/motr/c2429b70323846f0a44052f9a6572a24/addb... OK (0m16s)
Please attach these files to the bug report:
  m0reportbug-data.tar.gz
  m0reportbug-cores.tar.gz
  m0reportbug-traces.tar.gz
[root@cortx-data-headless-svc-ssc-vm-g4-rhev4-1293 ~]# cd /var/cortx/support_bundle/bundle-logs-cortx-data-ssc-vm-g4-rhev4-1209-6554467f46-qc2d4-2022-03-25_00-17/motr/
[root@cortx-data-headless-svc-ssc-vm-g4-rhev4-1293 motr]# ls
motr_bundle-logs-cortx-data-ssc-vm-g4-rhev4-1209-6554467f46-qc2d4-2022-03-25_00-17_7f4f141538794027a07e56de77107aa8_motr.tar.gz
[root@cortx-data-headless-svc-ssc-vm-g4-rhev4-1293 motr]# tar -xvf motr_bundle-logs-cortx-data-ssc-vm-g4-rhev4-1209-6554467f46-qc2d4-2022-03-25_00-17_7f4f141538794027a07e56de77107aa8_motr.tar.gz
cortx-data-headless-svc-ssc-vm-g4-rhev4-1293/
cortx-data-headless-svc-ssc-vm-g4-rhev4-1293/_m0reportbug-bundler.stderr
cortx-data-headless-svc-ssc-vm-g4-rhev4-1293/m0reportbug-data.tar.gz
cortx-data-headless-svc-ssc-vm-g4-rhev4-1293/m0reportbug-cores.tar.gz
cortx-data-headless-svc-ssc-vm-g4-rhev4-1293/m0reportbug-traces.tar.gz

**Review by cdeshmukh — APPROVED** [review]

**Review by yatin-mahajan — APPROVED** [review]
Looks good to me

**Review by VinothVijayakumar-dev — APPROVED** [review]

**Pull request merged**